code/game/objects/items/storage/briefcase.dm

```diff
@@ -48,3 +48,24 @@
 	new /obj/item/ammo_box/magazine/sniper_rounds/soporific(src)
 	new /obj/item/suppressor/specialoffer(src)
 
+/obj/item/storage/briefcase/case
+	name = "case"
+	icon_state = "case"
+	force = 2
+	w_class = WEIGHT_CLASS_NORMAL
+
+/obj/item/storage/briefcase/case/lobotomy
+	name = "lobotomy kit"
+
+/obj/item/storage/briefcase/case/lobotomy/PopulateContents()
+	new /obj/item/surgical_drapes(src)
+	new /obj/item/orbitoclast(src)
+	new /obj/item/hammer(src)
+	new /obj/item/cautery(src)
+	new /obj/item/reagent_containers/glass/bottle/morphine(src)
+	new /obj/item/paper/guides/lobotomy(src)
+	new /obj/item/healthanalyzer(src)
+
+/obj/item/paper/guides/lobotomy
+	name = "Lobotomies For Dummies"
+	info = "Doing a Lobotomy is easy.<br> Target the eyes, use your drapes, orbitoclast, hammer, orbitoclast, then finish off with a cautery.<br> However, to retrieve special powers from this is a little harder.<br> You must fail the lobotomizing step (second orbitoclast) to cause some traumas.<br> If these traumas are too bad, do the step correctly (Removes Severe Traumas by just cutting off more of the Brain) then fail it again.<br> To fail a step on purpose, you must be on disarm intent."
```

Comment:
SUCCEEDING on a lobotomy will also apply a trauma.

also, correct me if I'm wrong, but I don't think that succeeding on a lobotomy step actually heals any brain traumas, which is probably important to note, especially seeing as how failing a lobotomy does 80 brain damage.

Reply:
Sorry, this is outdated information. I'll rewrite it.